scripts/token-hud-extension.js

```diff
@@ -136,6 +140,49 @@ class TokenHudExtension {
 
    }
 
+   
+   static async addArmorTokenTip(app, html, data, actor) {
+
+        if (game.scenes.active.isView === false)
+        return;
+
+        const actorCharacteristics = actor.data.data.characteristics;
+        const ap = actor.prepareItems();
+        let toughness = actorCharacteristics.t.value;
+        let shieldAp = (ap.AP.shield - ap.totalShieldDamage);
+
+        let TooltipAp = game.i18n.localize('CHAR.T') + ": " + toughness +"; " + game.i18n.localize("SHEET.ShieldAP") + ": " + (ap.AP.shield - ap.totalShieldDamage) + "&#10;";
+        TooltipAp += game.i18n.localize("SHEET.ArmourAP") + " (" + ap.AP.head.label + "): " + ap.AP.head.value + "; " + game.i18n.localize("SHEET.ArmourAP") + " (" + ap.AP.body.label + "): " + ap.AP.body.value + "&#10;";
+        TooltipAp += game.i18n.localize("SHEET.ArmourAP") + " (" + ap.AP.rArm.label + "): " + ap.AP.rArm.value + "; " + game.i18n.localize("SHEET.ArmourAP") + " (" + ap.AP.lArm.label + "): " + ap.AP.lArm.value + "&#10;";
+        TooltipAp += game.i18n.localize("SHEET.ArmourAP") + " (" + ap.AP.rLeg.label + "): " + ap.AP.rLeg.value + "; " + game.i18n.localize("SHEET.ArmourAP") + " (" + ap.AP.lLeg.label + "): " + ap.AP.lLeg.value;
+        // Create space for Hud Extensions next to status icon
+        let divTokenHudExt = '<div class="tokenhudext right">';
+
+        html.find('.control-icon.effects').wrap(divTokenHudExt);
+        let hudAp = $('<div class="control-icon tokenhudicon right" title="' + TooltipAp + '"><i class="fas fa-shield-alt"></i> ' + toughness + "/" + shieldAp + '</div>');
+        html.find('.control-icon.effects').after(hudAp);
+        hudAp.find('i').dblclick(async (ev) => {
+            if (ev.ctrlKey && ev.shiftKey) {
+                if (hasSkill(actor, "Dodge") !== null) {
```

Comment:
Dodge is already available as a shortcut on Movement, so we shouldn't duplicate that on Armour.